torch_xla/csrc/elementwise.cpp

```diff
@@ -66,8 +66,16 @@ xla::XlaOp BuildThreshold(xla::XlaOp input, xla::XlaOp output,
 
 xla::XlaOp BuildRelu(xla::XlaOp input) {
   const xla::Shape& input_shape = ShapeHelper::ShapeOfXlaOp(input);
+#ifndef EXPERIMENTAL_XLA_UNBOUNDED_DYNAMISM
```

Comment:
Let's also use env var instead of macros? Macros need to recompile if changed.

Reply:
+1

Reply:
Sounds good, will update to use the env var. The intension was to enable this experimental feature at compile time, so it won't affect the default. lowering workflow